### PR TITLE
Added UI enhancements to the preview page

### LINKED ISSUE #4246: Improve File Preview UI
I see a lot of scope for improvement in the below screenshot

image

### PR DESCRIPTION
**Pull request by yogeshbhutkar**
### <samp>🤖 Generated by Copilot at dd1b719</samp>

This file enhances the file preview dialog component by using a custom button, a smoother transition, and a better close button. These changes improve the user interface and experience of viewing files in the app.

## Proposed Changes

- Fixes #4246 
- Added ButtonV2 Care UI components to make the design look consistent.
- Gave the close button a modern look.
- Removed the old school ease animation and added an aesthetic bottom to top moving, translate-y animation to make more sense from UI's perspective.

An image of the new tested solution.
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### <samp>🤖 Generated by Copilot at dd1b719</samp>

*  Replace native `button` elements with custom `ButtonV2` component in file preview dialog ([link](https://github.com/coronasafe/care_fe/pull/6221/files?diff=unified&w=0#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2R7), [link](https://github.com/coronasafe/care_fe/pull/6221/files?diff=unified&w=0#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2L147-R155), [link](https://github.com/coronasafe/care_fe/pull/6221/files?diff=unified&w=0#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2L173-R175))
*  Modify transition and style of file preview dialog to slide up from bottom and have rounded top border and shadow ([link](https://github.com/coronasafe/care_fe/pull/6221/files?diff=unified&w=0#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2L70-R92))
*  Remove backdrop blur and black overlay from file preview dialog ([link](https://github.com/coronasafe/care_fe/pull/6221/files?diff=unified&w=0#diff-54c239f5693dcce454bb02b533a03b41fc9d4c5998bf3b1ce0aef9fafbe5ceb2L70-R92))

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
1. There are two shadows!
2. The buttons make sense to be placed center bottom maybe?

**Comment by rithviknishad** [comment]
@yogeshbhutkar consider using the `Slideover` component itself and pass `slideFrom` prop with "bottom" to re-use existing components!

**Comment by yogeshbhutkar** [comment]
Patch 1

- Removed duplicate shadows.
- Centered the buttons.

Related Screenshot:
image

**Comment by rithviknishad** [comment]
@yogeshbhutkar is this ready for review?

Just curious to know, was there a reason why you haven't reused the existing slideover component?

**Comment by rithviknishad** [comment]
Also, you'll have to untrack ur package-lock.json changes as you are not changing any dependencies voluntarily

**Comment by yogeshbhutkar** [comment]
Here's a SS for the newly migrated to SlideOver component preview page. 

image

File: src/Components/Common/FilePreviewDialog.tsx [hunk 1]
```diff
@@ -66,153 +56,107 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
...
-                  ) : (
-                    <div className="flex h-screen items-center justify-center">
-                      <div className="text-center">
-                        <CircularProgress />
-                      </div>
-                    </div>
-                  )}
-                </Dialog.Panel>
-              </Transition.Child>
+                    <i className="fas fa-download mr-2" />
```
Comment (rithviknishad) [inline src/Components/Common/FilePreviewDialog.tsx]: You should be using CareIcons in newer implementations. We are phasing out usage of font-awesome icons (all solid icons especially)

File: src/Components/Common/FilePreviewDialog.tsx [hunk 2]
```diff
@@ -66,153 +56,107 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
...
-            leaveFrom="opacity-100"
-            leaveTo="opacity-0"
-          >
-            <div className="fixed inset-0 bg-black/50 backdrop-blur-sm transition-all" />
-          </Transition.Child>
+  const [, setOpen] = useState(false);
+
+  // const toggleSetOpen = () => {
+  //   setOpen((prev) => !prev);
+  // };
```
Comment (rithviknishad) [inline src/Components/Common/FilePreviewDialog.tsx]: Was this commented for any purpose? If not used feel free to delete it off.

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: You've deleted package-lock file instead of un-tracking the changes.

Revert changes made to this file.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Everything else LGTM! Nice work!

File: src/Components/Common/FilePreviewDialog.tsx (hunk 1 above)
Comment (yogeshbhutkar) [reply src/Components/Common/FilePreviewDialog.tsx]: Sure.

**Comment by rithviknishad** [comment]
Closed in favour of #6296

**Pull request closed without merging**